skimage/filters/_unsharp_mask.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+from scipy.ndimage.filters import gaussian_filter
+from skimage.util.dtype import dtype_limits
+
+
+def _unsharp_mask_single_channel(image, radius, amount):
+    """Single channel implementation of the unsharp masking filter."""
+    blurred = gaussian_filter(image.astype(np.float_),
+                              sigma=radius,
+                              mode='reflect')
+    result = image + (image - blurred) * amount
+
+    # Enforce image dtype limits
+    if image.dtype.kind == 'f':
```

Comment:
Should probably be `np.issubdtype(image.dtype, float)`.

Reply:
Ping on this one.

Reply:
@stefanv could you describe the difference between these two alternatives?